src/price.rs

```diff
@@ -7,10 +10,24 @@ impl Contract {
         self.prices.insert(&token_id, &price);
     }
 
-    #[private]
     pub fn view_price(&self, token_id: AccountId) -> Price {
```

Comment:
move this method to view.rs and add get_price here and get price must return just value